### ISSUE TITLE
Stack Overflow Question/Answer(s) of the week section

### ISSUE THREAD

**Issue by amrrs** [rweekly/rweekly.org#373]
Was wondering if this would help others - either a link or a section with SO Q&As.

Thx!
Abd

**Comment by maelle** [comment]
Wouldn't it be quite hard to maintain? 🤔 And there's the RStudio community forum as well...

I tend to prefer blog posts, but I see that it'd be a section similar to the Twitter quotes of the week.

Let's see what others say!

**Comment by jonmcalder** [comment]
Sorry for the delayed reply. Just wanted to say that I quite like this idea. As @maelle says, I'm not entirely convinced about it's long term viability, but I can certainly see how highlighting an interesting StackOverflow Q&A every now and then could be a useful learning resource. I remember @karawoo doing an occasional segment like this with @hugobowne for DataFramed?

**Comment by hugobowne** [comment]
Correct! e.g. ~17:15 in [this episode](https://www.datacamp.com/community/podcast/data-science-telecommunications).

I'm always super happy to hear pitches for future segments.

**Comment by qinwf** [comment]
There are some Q&A links in the past issues.

https://github.com/rweekly/rweekly.org/search?l=Markdown&q=StackOverflow.com

 It is Ok to add thees links. And also not limited to StackOverflow.
